### PR TITLE
Puppeteer core dependency

### PR DESCRIPTION
**Pull request by sbounmy**
when using remote chromium `browserless/chrome`, `puppeteer-core` is more lightweight and avoid download chrome.

### CODE REVIEW

**Review by rud — APPROVED** [review]
Sounds like an excellent plan! 

An idea: Maybe make it a config option so one can explicitly pick whether `puppeteer` or `puppeteer-core` is the command to invoke?

Note: I'm not affiliated with the project, but this seems like a good plan.

**Comment by sbounmy** [comment]
@rud IMHO its better to not have another config option.
Most people have puppeteer or puppeteer-core installed so the gem should be ale to pick which one is available.
`puppeteer` or fallback to `puppeteer-core`

**Comment by sbounmy** [comment]
@abrom any update for merging this ?

File: spec/grover/processor_spec.rb [hunk 1]
```diff
@@ -95,6 +101,23 @@
...
+          FileUtils.move 'node_modules/puppeteer_temp', 'node_modules/puppeteer'
+        end
+
+        let(:options) { { 'browserWsEndpoint' => browser_ws_endpoint } }
+        let(:browser_ws_endpoint) { 'ws://localhost:3000/' }
+        let(:url_or_html) { '<html><body style="background-color: blue"></body></html>' }
+
+        it 'does not raise' do
+          expect { convert }.to_not raise_error
+        end
```
Comment (abrom) [inline spec/grover/processor_spec.rb]: Specs look good, but I think we could do better with one that also checks that the result actually was a PDF with the expected content. Something like:

```suggestion
        let(:url_or_html) { '<html><body style="background-color: blue">puppeteer-core works!</body></html>' }

        it { expect { convert }.not_to raise_error }
        it { expect(pdf_text_content).to eq 'puppeteer-core works!' }
```

File: spec/grover/processor_spec.rb [hunk 2]
```diff
@@ -95,6 +101,23 @@
         end
       end
 
+      context 'when puppeteer-core package only is installed', :remote_browser do
+        before do
+          FileUtils.move 'node_modules/puppeteer', 'node_modules/puppeteer_temp'
+        end
+        after do
+          FileUtils.move 'node_modules/puppeteer_temp', 'node_modules/puppeteer'
+        end
```
Comment (abrom) [inline spec/grover/processor_spec.rb]: Rubocop is either going to want a newline between these, or use the inline block form:
```suggestion
        before { FileUtils.move 'node_modules/puppeteer', 'node_modules/puppeteer_temp' }
        after { FileUtils.move 'node_modules/puppeteer_temp', 'node_modules/puppeteer' }
```

**Review by abrom — APPROVED** [review]
Nice one @sbounmy looks great! I've flagged a few minor things but I've pushed a commit to address them 😄

**Comment by abrom** [comment]
Released in [v1.1.7](https://rubygems.org/gems/grover/versions/1.1.7)

**Pull request merged**